conda_devenv/devenv.py

```diff
@@ -456,10 +456,12 @@ def get_env_directory(env_name):
     import json
     info = subprocess.check_output(["conda", "info", "--json"]).decode()
     info = json.loads(info)
-    envs = info["envs"]
+    envs_dirs = info["envs_dirs"]
 
-    for env in envs:
-        if os.path.basename(env) == env_name:
+    for directory in envs_dirs:
+        env = os.path.join(directory, env_name)
+        conda_meta_dir = os.path.join(env, 'conda-meta')
+        if os.path.isdir(conda_meta_dir) and os.path.basename(env) == env_name:
```

Comment:
I think that `os.path.basename(env) == env_name` is redundant due `env = os.path.join(directory, env_name)`.